Ethereum/ethscriptions/project.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  EthereumProject,
+  EthereumDatasourceKind,
+  EthereumHandlerKind,
+} from "@subql/types-ethereum";
+
+// Can expand the Datasource processor types via the generic param
+const project: EthereumProject = {
+  specVersion: "1.0.0",
+  version: "0.0.1",
+  name: "ethscriptions",
+  description: "This SubQuery project indexes inscriptions on Ethereum Mainnet",
+  runner: {
+    node: {
+      name: "@subql/node-ethereum",
+      version: ">=3.0.0",
+    },
+    query: {
+      name: "@subql/query",
+      version: "*",
+    },
+  },
+  schema: {
+    file: "./schema.graphql",
+  },
+  network: {
+    /**
+     * chainId is the EVM Chain ID, for Ethereum this is 1
+     * https://chainlist.org/chain/1
+     */
+    chainId: "1",
+    /**
+     * These endpoint(s) should be public non-pruned archive node
+     * We recommend providing more than one endpoint for improved reliability, performance, and uptime
+     * Public nodes may be rate limited, which can affect indexing speed
+     * When developing your project we suggest getting a private API key
+     * If you use a rate limited endpoint, adjust the --batch-size and --workers parameters
+     * These settings can be found in your docker-compose.yaml, they will slow indexing but prevent your project being rate limited
+     */
+    endpoint: ["https://rpc.ankr.com/eth"],
+  },
+  dataSources: [
+    {
+      kind: EthereumDatasourceKind.Runtime,
+      startBlock: 18130011,
+      mapping: {
+        file: "./dist/index.js",
+        handlers: [
+          {
+            kind: EthereumHandlerKind.Call,
+            handler: "handleTransaction",
```

Comment:
I haven't tested this but if you filter the function this should greatly speed up indexing by being able to use dictionaries. 

The function in this case is `data:` encoded in hex

```suggestion
            handler: "handleTransaction",
            filter: {
              function: "0x64617461"
            }
```

Reply:
Please test and include if working